content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+ 900   708   svchost.exe           x64   0        NT AUTHORITY\NETWORK SERVICE  
+ 948   708   svchost.exe           x64   0        NT AUTHORITY\LOCAL SERVICE    
+ 1016  660   LogonUI.exe           x64   1        NT AUTHORITY\SYSTEM           C:\Windows\system32\LogonUI.exeost.exe           x64   0        NT AUTHORITY\LOCAL SERVICE    
+ 1172  708   svchost.exe           x64   0        NT AUTHORITY\NETWORK SERVICE  
+ 1304  708   spoolsv.exe           x64   0        NT AUTHORITY\SYSTEM           C:\Windows\System32\spoolsv.exe
+ 1340  708   svchost.exe           x64   0        NT AUTHORITY\LOCAL SERVICE    
+ 1408  708   amazon-ssm-agent.exe  x64   0        NT AUTHORITY\SYSTEM           C:\Program Files\Amazon\SSM\amazon-ssm-agent.exe
+ 1480  708   LiteAgent.exe         x64   0        NT AUTHORITY\SYSTEM           C:\Program Files\Amazon\XenTools\LiteAgent.exe
+ 1616  708   Ec2Config.exe         x64   0        NT AUTHORITY\SYSTEM           C:\Program Files\Amazon\Ec2ConfigService\Ec2Config.exe
+ 1956  708   svchost.exe           x64   0        NT AUTHORITY\NETWORK SERVICE  
+ 2060  708   SearchIndexer.exe     x64   0        NT AUTHORITY\SYSTEM           
+ 2072  832   WmiPrvSE.exe                                                       
+ 2184  708   svchost.exe           x64   0        NT AUTHORITY\LOCAL SERVICE    
+ 2220  708   mscorsvw.exe          x64   0        NT AUTHORITY\SYSTEM           C:\Windows\Microsoft.NET\Framework64\v4.0.30319\mscorsvw.exe
+ 2496  708   TrustedInstaller.exe  x64   0        NT AUTHORITY\SYSTEM           
+ 2548  708   sppsvc.exe            x64   0        NT AUTHORITY\NETWORK SERVICE  
+ 2956  708   vds.exe               x64   0        NT AUTHORITY\SYSTEM           
+ 2996  708   svchost.exe           x64   0        NT AUTHORITY\SYSTEM
+```
+
+Luckily for me it already looked like it was in the injected in the spool service so there was no need to proceed with a migration.
+
+While I already knew where this host was running from deploying it, this `ps` output also reiterated that this host was an EC2 instance via the `ssm-agent`, `LiteAgent` and `Ec2Config` processes. While this could also be seen in the nmap scans via the reverse dns lookup, I thought it was still worth calling out.
```

Comment:
Even though I already knew where this host was running from deploying it, the `ps` output also established that this host was an EC2 instance via the `ssm-agent`, `LiteAgent` and `Ec2Config` processes. This could have also been observed in the nmap scans via the reverse dns lookup, but I thought it was worth calling out.